pkg/ui/ui.go

```diff
@@ -108,14 +85,14 @@ func (bu *BaseUI) getAssetFile(filename string) (os.FileInfo, []byte, error) {
 	return info, file, nil
 }
 
-func (bu *BaseUI) serveAsset(fp string, w http.ResponseWriter, req *http.Request) {
+func (bu *BaseUI) serveAsset(fp string, w http.ResponseWriter, req *http.Request) error {
 	info, file, err := bu.getAssetFile(fp)
 	if err != nil {
 		level.Warn(bu.logger).Log("msg", "Could not get file", "err", err, "file", fp)
-		w.WriteHeader(http.StatusNotFound)
```

Comment:
Move these two lines in the error check after line 42 as mentioned in the previous comment.

Reply:
Thanks for your review , i made the changes you suggested.